components/style/properties/properties.mako.rs

```diff
@@ -2560,6 +2576,10 @@ impl<'a> StyleBuilder<'a> {
             device,
             parent_style,
             inherited_style,
+            // None of our callers pass in ::first-line parent styles.
```

Comment:
Hmm, I wonder if this is true when you animate an element inside a `::first-line`...